File: example/typescript/class/final-variable.ts
class Point {
  readonly _x
  readonly _y
  constructor(x, y) {
    this._x = x
    this._y = y
  }
  // Point类的剩余代码 ...
}

const originPoint = new Point(0, 0)

// originPoint = new Point(0, 0) // Cannot assign to 'originPoint' because it is a constant.

File: example/dart/class/final-variable.dart
class Point {
  final x, y;
  Point(this.x, this.y);
  // Point类的剩余代码 ...
}

final originPoint = new Point(0, 0);

main(List<String> args) {
  // originPoint = new Point(0, 0); // 'originPoint', a final variable, can only be set once.
}